ANS_JS_template.tex

```diff
@@ -93,84 +93,96 @@
 
+Our literature review revealed that a lot of work on NRHES is in the design and analysis stage \cite{Boardman2013, Shropshire2012}. Research is focused on developing a model that can dynamically simulate the contributions of variable energy sources, the constraints of fluctuating electricity demand, and thermal/electrical power demands for various industrial processes. The models focus on answering questions regarding minimizing costs for each of the components comprising the NRHES. In order to move forward with development of any NRHES,  such systems must demonstrate that they can be profitable and can reliably respond to fluctuating electricity production and demand \cite{Rabiti2015}.
 
-\subsection{Computational Tools}
-NRHES sit in an interesting location in computational modeling. Many tools have been developed to model nuclear reactors and the nuclear fuel cycle as demonstrated in the nuclear fuel cycle simulators shown in Figure 2. Many tools have also been created to model and optimize non-nuclear hybrid energy systems (HOMER, Hybrid2, SOLSIM, SOMES, ARES, RAPSIM, HOGA)\cite {Bernal-Agustin2009}. These programs generate mass flow and calculate electricity output and the possible output of another product, such as hydrogen. Modelica and Excel, while neither specifically HES or NRHES tools, have been used to model both nuclear and non-nuclear hybrid energy systems \cite{Shropshire2012, Chen2016, Binder2014, Garcia2015, Epiney2016}. HES tools are typically bought for microgrid standalone systems. The main challenges to applying HES systems to a NRHES are the scale of the system being modeled and including the grid demand.
+An ongoing effort at Idaho National Laboratory (INL), Argonne National Laboratory (ANL), and Oak Ridge National Laboratory (ORNL) focuses on modeling a generic NRHES. The author has worked with some input from researchers working on the NRHES model. The generic model will test the economic viability of a NRHES to determine how future development should be pursued. The industrial plant combines a nuclear power plant, a wind farm, a natural gas power plant, a battery, and a high temperature steam electrolysis hydrogen production facility. More component models will be developed in the future that can allow for the comparison of various industrial customers\cite{Harrison2016}. The chosen renewable source, a wind farm, models a highly stochastic system due to the unpredictable nature of wind generation \cite{Chen2016_wind}. The model optimizes the sizes of the nuclear power plant, the natural gas plant, and the battery. The objective is to choose the NRHES configuration that minimizes costs while meeting set emissions and grid reliability expectations.
```

Comment:
Your "the author statement" is clunky. You can say "As part of this study, we have worked with INL researchers in the X Division who are leading experts in the field."

Reply:
I haven't really worked with them though.